debezium-connector-jdbc/README.md

```diff
@@ -0,0 +1,152 @@
+These relational model classes can be found in the `io.debezium.connector.jdbc.relational` package.
+
+## Building the JDBC sink connector
+
+The following is required in order to work with the Debezium JDBC sink connector code base, and to build it locally:
+
+* [Git](https://git-scm.com) 2.2.1 or later
+* JDK 17 or later, e.g. [OpenJDK](http://openjdk.java.net/projects/jdk)
+* [Docker Engine](https://docs.docker.com/engine/install/) or [Docker Desktop](https://docs.docker.com/desktop/) 1.9 or later
+* [Apache Maven](https://maven.apache.org/index.html) 3.9.8 or later
+  (or invoke the wrapper with `.mvnw` for Maven commands)
+
+### Why Docker?
+
+The test suite is heavily based on TestContainer usage, and automatically starts a variety of source and sink databases automatically.
+Without a Docker-compatible environment, the integration tests will not run.
+If you don't have a Docker environment, you can skip the integration tests by using the `-DskipITs` command line argument, shown below:
+
+    $ ./mvnw clean verify -DskipITs
+
+### Running the tests
+
+There are three types of types in the test suite:
+
+* Unit tests
+* Sink-based integration tests
+* End to end matrix-based Integration tests
+
+By default all unit tests are executed as a part of the build.
+The sink-based integration tests are only executed for MySQL, PostgreSQL, and SQL Server by default, while none of the end-to-end matrix-based tests are executed.
+
+In order to execute the sink-based integration tests for Oracle and DB2, the `-Dtest.tags` argument must be provided to include these in the build.
+In order to do this, add all the integration tests to be executed, as shown below for all databases:
+
+    $ ./mvnw clean install -Dtest.tags=it-mysql,it-postgresql,it-sqlserver,it-oracle,it-db2
+
+In order to run all sink-based integration tests for all databases, a short-cut tag is provided:
+
+    $ ./mvnw clean install -Dtest.tags=it
+
+Similarly, in order to enable specific end to end tests, the `-Dtest.tags` argument can also be supplied with the necessary tags for each sink database type:
+
+    $ ./mvnw clean install -Dtest.tags=e2e-mysql,e2e-postgresql,e2e-sqlserver,e2e-oracle,e2e-db2
+
+In order to run all end to end integration tests, a short-cut tag is provided as well:
+
+    $ ./mvnw clean install -Dtest.tags=e2e
+
+In order to run all tests for all source/sink combinations:
+
+    $ ./mvnw clean install -Dtest.tags=all
+
+## Contributing
+
+The Debezium community welcomes anyone that wants to help out in any way, whether that includes reporting problems, helping with documentation, or contributing code changes to fix bugs, add tests, or implement new features.
+See [this document](https://github.com/debezium/debezium/blob/main/CONTRIBUTE.md) for details.
+
+A big thank you to all the Debezium JDBC sink contributors!
+
+<a href="https://github.com/debezium/debezium-connector-jdbc/graphs/contributors">
```

Comment:
As the project is now part of main Debezium repo, this list is possibly obsolete.

Reply:
Great catch, cut-n-paste failure 😁